### PR TITLE
Improve English documentation

### PR DESCRIPTION
**Pull request by nikogdanikomu**

### CODE REVIEW

File: docs/pages/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ pagination_label: Code of Conduct
 
 ## Our Pledge
 
-We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+As members, contributors and leaders, we pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (afuno) [inline docs/pages/CODE_OF_CONDUCT.md:13]: These changes need to be reverted.

Comment (nikogdanikomu) [reply docs/pages/CODE_OF_CONDUCT.md:13]: ok, boss

File: docs/pages/usage/failures.md [hunk 2]
```diff
@@ -8,39 +8,36 @@ pagination_label: Service failures
 
 # Failures
 
-The methods that are used in `make` may fail. In order to more informatively provide information about this outside the service, the following methods were prepared.
+In a simple use case, all service failures (or crashes) will come from input, output or internal attributes.
```
Comment (afuno) [inline docs/pages/usage/failures.md:11]: I think in the english version we can skip "(or crashes)". This clarification was appropriate in the Russian version.

File: docs/pages/usage/result.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ pagination_label: Service result
 
 # Result
 
-All services have the result of their work. For example, in case of success this call:
+All services have the result of their work. For instance, in case of success this call:
```
Comment (afuno) [inline docs/pages/usage/result.md:11]: It's better to use the word "example" here.

Comment (nikogdanikomu) [reply docs/pages/usage/result.md:11]: Oh yes, thank you very much, coach

File: docs/pages/usage/failures.md (hunk 2 above)
Comment (nikogdanikomu) [reply docs/pages/usage/failures.md:11]: Yes, you're absolutely right, boss. Only russian version needs this clarification

File: docs/pages/usage/output.md [hunk 4]
```diff
@@ -8,6 +8,16 @@ pagination_label: Service output attributes
 
 # Output
 
+All attributes the service must return in `Result` should be described through the `output` method.
+
+## Options
+
+### `Type` option
```
Comment (afuno) [inline docs/pages/usage/output.md:15]:
```md
### Option `type`
```

File: docs/pages/usage/internal.md [hunk 5]
```diff
@@ -8,6 +8,16 @@ pagination_label: Service internal attributes
 
 # Internal
 
+Internal private attributes can be defined through the internal method.
+
+## Options
+
+### `Type` option
```
Comment (afuno) [inline docs/pages/usage/internal.md:15]:
```md
### Option `type`
``

File: docs/pages/usage/input.md [hunk 6]
```diff
@@ -195,7 +219,7 @@ input :first_name,
       }
 ```
 
-### Option `inclusion`
+### Опция `inclusion`
```
Comment (afuno) [inline docs/pages/usage/input.md:222]: Option

File: docs/pages/usage/input.md [hunk 7]
```diff
@@ -220,7 +244,7 @@ input :event_name,
 
 :::note
 
-The `must` option can only work in advanced mode.
+`must` option can work only in advanced mode.
```
Comment (afuno) [inline docs/pages/usage/input.md:247]:
```md
The `must` option can work only in advanced mode.
```

**Pull request merged**